compiler/ras/Debug.cpp

```diff
@@ -1601,6 +1601,8 @@ TR_Debug::getName(TR::SymbolReference * symRef)
             return "<arraycmp>";
          case TR::SymbolReferenceTable::currentTimeMaxPrecisionSymbol:
             return "<currentTimeMaxPrecision>";
+         case TR::SymbolReferenceTable::encodeASCIISymbol:
+            return "<encodeASCIISymbol>";
```

Comment:
Not a big deal at all, but the name here and in the table below can omit the `Symbol` suffix like most of the other "non-helper" symbols do